<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenCVWindow</class>
 <widget class="QDialog" name="OpenCVWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabOriginal">
      <attribute name="title">
       <string>Original Image</string>
      </attribute>
      <widget class="QLabel" name="labelOriginal">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>711</width>
         <height>361</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEditFlag">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>390</y>
         <width>321</width>
         <height>121</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tabProcessed">
      <attribute name="title">
       <string>Processed Image</string>
      </attribute>
      <widget class="QLabel" name="labelProcessed">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>841</width>
         <height>431</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
